### ISSUE TITLE
Cannot npm install

### ISSUE THREAD

**Issue by anterpin** [graphcommerce-org/graphcommerce#1746]
### Provide environment information

- `node -v`:
8.19.2
- `npm -v`
8.19.2
- `uname -a`:
Linux manjaro 5.10.154-1-MANJARO \# 1 SMP PREEMPT Thu Nov 10 20:51:28 UTC 2022 x86_64 GNU/Linux

### Describe the Bug

On `npm install`
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @graphcommerce/demo-magento-graphcommerce@5.0.0
npm ERR! Found: @mui/material@5.10.0
npm ERR! node_modules/@mui/material
npm ERR!   @mui/material@"5.10.0" from the root project
npm ERR!   peer @mui/material@"5.10.0" from @graphcommerce/googleanalytics@5.0.0
npm ERR!   node_modules/@graphcommerce/googleanalytics
npm ERR!     @graphcommerce/googleanalytics@"5.0.0" from the root project
npm ERR!   5 more (@graphcommerce/magento-wishlist, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @mui/material@"5.5.3" from @graphcommerce/demo-magento-graphcommerce@5.0.0
npm ERR! node_modules/@graphcommerce/demo-magento-graphcommerce
npm ERR!   @graphcommerce/demo-magento-graphcommerce@"5.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @mui/material@5.5.3
npm ERR! node_modules/@mui/material
npm ERR!   peer @mui/material@"5.5.3" from @graphcommerce/demo-magento-graphcommerce@5.0.0
npm ERR!   node_modules/@graphcommerce/demo-magento-graphcommerce
npm ERR!     @graphcommerce/demo-magento-graphcommerce@"5.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Expected Behavior

It should install

### To Reproduce

1. `git clone -b main --depth 1 https://github.com/graphcommerce-org/graphcommerce.git`
2. `mkdir my-project`
3. `cp -R graphcommerce/examples/magento-graphcms/. my-project`
4. `cd my-project`
5. `cp -R .env.example .env`
6. `rm CHANGELOG.md`
7. `rm -r node_modules && rm -r .next`
8. `npm install`

**Comment by paales** [comment]
Can you try with `yarn`. I'm out of the office for a few days, but will try to take a look next week.

**Comment by jesse-stewart** [comment]
@paales - you are a hero helping people while OoO

**Comment by anterpin** [comment]
With `yarn` it works. Thank you.

**Comment by paales** [comment]
Fixing it in https://github.com/graphcommerce-org/graphcommerce/pull/1752

**Comment by paales** [comment]
Should be fixed in 5.1.0 https://github.com/graphcommerce-org/graphcommerce/releases/tag/release-1671034410823
